convert_color_scheme.py

```diff
@@ -0,0 +1,135 @@
+"""Functionality for converting a color scheme to a "TerminalView" scheme."""
+import os
+import plistlib
+import sublime
+
+
+def hex_to_rgb(hexstring):
+    """Convert a string representing a hex color to an RGB tuple."""
+    return (int(hexstring[1:3], 16) / 255,
+            int(hexstring[3:5], 16) / 255,
+            int(hexstring[5:7], 16) / 255)
+
+
+def rgb_to_hex(rgb):
+    """Convert an RGB tuple to a hex string."""
+    return "#{:02x}{:02x}{:02x}".format(int(rgb[0] * 255),
+                                        int(rgb[1] * 255),
+                                        int(rgb[2] * 255))
+
+
+def norm2(vec3):
+    """Compute the squared norm of a three-dimensional vector."""
+    return vec3[0]**2 + vec3[1]**2 + vec3[2]**2
+
+
+def distance2(a, b):
+    """Compute the squared distance between two 3D vectors."""
+    return norm2((a[0] - b[0], a[1] - b[1], a[2] - b[2]))
```

Comment:
<img width="585" alt="schermafbeelding 2017-08-13 om 22 08 00" src="https://user-images.githubusercontent.com/2431823/29253031-ea4dfee6-8073-11e7-89cb-e9b8bb9589a6.png">
I get something different now when I use the cie94 metric